src/Components/GameInfo.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import {GameInfoProps} from '../Types/TypeProps'
+
+/**
+ * Render Game Info and Game toolbar buttons
+ * @param param0 
+ * @returns 
+ */
+const GameInfo: React.FC<GameInfoProps> = (props) => {
+    return (
+        <>
+            <span style={{display: props.gameWinner ? 'block' : "none"}} className="game-info-status-winner">{props.gameWinner}</span>
+            <span style={{display: props.status && !props.isGameOver() ? 'block' : 'none'}} className="game-info-status">{props.status}</span>
```

Comment:
"isGameOver" seems like too much abstraction making it into a function. Instead it can just be a boolean and where you use GameInfo you just send the condition(boolean) to "isGameOver"